src/main/java/de/unistuttgart/overworldbackend/data/MinigameTask.java

```diff
@@ -23,6 +23,7 @@ public class MinigameTask {
   int index;
   String game;
   UUID configurationId;
+  String description;
```

Comment:
```suggestion

  @Nullable
  String description;
```

Reply:
ok, I thought it would just be necessary on DTOs, but I will use yours now

Reply:
It is, but I wanted to be consistent.